AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md

```diff
@@ -53,48 +53,47 @@ You will need to download and install the following toolkits, tools, and compone
 <br>If you have not already, select and install these Tools via [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html). AI and Analytics samples are validated on AI Tools Offline Installer. It is recommended to select Offline Installer option in AI Tools Selector.<br>
-or simple pip install in your current ready python environment 
-```
-pip install tensorflow==2.14
-```
-please see the[supported versions](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html).
+please see the [supported versions](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html).
 
-## Run the Sample
+>**Note**: If Docker option is chosen in AI Tools Selector, refer to [Working with Preset Containers](https://github.com/intel/ai-containers/tree/main/preset) to learn how to run the docker and samples.
 
->**Note**: Before running the sample, make sure Environment Setup is completed.
-Go to the section which corresponds to the installation method chosen in [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to see relevant instructions:
-* [AI Tools Offline Installer (Validated)](#ai-tools-offline-installer-validated)
-* [Conda/PIP](#condapip) 
-* [Docker](#docker)
+**2. (Offline Installer) Activate the AI Tools bundle base environment**
 
-### AI Tools Offline Installer (Validated)  
-1. If you have not already done so, activate the AI Tools bundle base environment. If you used the default location to install AI Tools, open a terminal and type the following
+If the default path is used during the installation of AI Tools:
 ```
 source $HOME/intel/oneapi/intelpython/bin/activate
 ```
-If you used a separate location, open a terminal and type the following
+If a non-default path is used:
 ```
 source <custom_path>/bin/activate
 ```
-2. Activate the Conda environment:
+ 
+**3. (Offline Installer) Activate relevant Conda environment**
+
 ```
-conda activate tensorflow  
+conda activate tensorflow ## For the system with Intel CPU
+conda activate tensorflow-gpu ## For the system with Intel GPU  
 ``` 
-3. Clone the GitHub repository:
+
+**4. Clone the GitHub repository**
 ``` 
 git clone https://github.com/oneapi-src/oneAPI-samples.git
 cd oneAPI-samples/AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted
 ```
-### Run the Script
+## Run the Sample
 
-Run the Python script.
+>**Note**: Before running the sample, make sure Environment Setup is completed.
+Go to the section which corresponds to the installation method chosen in [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to see relevant instructions:
+* [AI Tools Offline Installer (Validated)](#ai-tools-offline-installer-validated)
+* [Conda/PIP](#condapip) 
+* [Docker](#docker)
+### Run the Script
 ```
 python TensorFlow_HelloWorld.py
 ```
```

Comment:
We still need a Docker section. GS samples are already included into AI preset containers, so we point users to preset containers repo
```suggestion
### Docker
AI Tools Docker images already have Get Started samples pre-installed. Refer to [Working with Preset Containers](https://github.com/intel/ai-containers/tree/main/preset) to learn how to run the docker and samples.
```